src/chainparams.cpp

```diff
@@ -682,7 +682,8 @@ class CDevNetParams : public CChainParams {
         assert(consensus.hashGenesisBlock == uint256S("0x000008ca1832a4baf228eb1553c03d3a2c8e02399550dd6ea8d65cec3ef23d2e"));
         assert(genesis.hashMerkleRoot == uint256S("0xe0028eb9648db56b1ac77cf090b99048a8007e2bb64b68f092c03c7f56a662c7"));
 
-        devnetGenesis = FindDevNetGenesisBlock(genesis, 50 * COIN);
+        devnetVersion = 1; // This should be bumped every time there is some breaking change to devnets
+        devnetGenesis = FindDevNetGenesisBlock(genesis, /*reward=*/50 * COIN, /*devnet_version=*/devnetVersion);
```

Comment:
Doesn't setting this to 1 mean that all devnets will be on version 1?

Could one devnet be set to 2 and another be set to 3 in the config, but here, that version is reset to 1?